### DISCUSSION TITLE
Jetson Nano Shuts down quickly - Battery Problem

### DISCUSSION THREAD

**Discussion by abhisheksreesaila** [prl-mushr/mushr#43] [Hardware]
Issue :  

- I use LiPo Batteries for powering JETSON Nano. 
- But Jetson Nano boots up for a few seconds and shuts down.  I made sure that i charged the battery to the near full capacity.  Converter shows 5V.   Any thoughts or ideas how to make the battery reliable? I am planning to change the step down converter to rule out the issue with the conversion.  Other than that, not sure what could be wrong with the setup. 

![connected to power source](https://user-images.githubusercontent.com/11252442/129236235-a8944b73-cc19-43c3-9cd9-05f1926ed71c.jpg)

![5Volts with battery charged](https://user-images.githubusercontent.com/11252442/129236220-7a349330-b48c-4d16-a530-aece9ec9978d.jpg)

**Comment by palanc** [comment]
Hi @abhisheksreesaila - two things to try:

1. Double check that there are no electrical shorts in any of the wires as well as at the sites where the wires enter both the input and output of the buck converter.
2. You could try slightly raising the output voltage of the buck converter (say to 5.1V). It is possible that when the Jetson Nano boots, the sudden required load current causes the buck converter's output voltage to momentarily drop below the Nano's minimum required voltage, so having it output a slightly higher voltage to compensate might allow it to successfully boot. Note that the Jetson Nano is spec'd to handle up to 5.25V, and also has an overvoltage protection circuit.

Let us know if those and/or replacing the buck converter help.

**Reply by abhisheksreesaila** [reply]
Thanks @planc509. This gives me some direction to work on. I will update you soon.

**Reply by abhisheksreesaila** [reply]
Yes it worked.  **I set the voltage 5.2** and its now way more reliable, and finally able to drive the car autonomously!
